packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -230,18 +177,28 @@ const ProgressIndicatorItemIcon = ({
 	isActive,
 }: {
 	status: ProgressIndicatorItemStatus;
-	isActive: boolean;
+	isActive?: boolean;
 }) => {
 	const { icon: Icon, iconColor } = statusMap[status];
 	const ringWidth = tokens.borderWidth.md;
 	const ringGap = 3;
 	const ringInset = ringWidth + ringGap;
-	const processedIconColor: IconColour =
+	const processedIconColor =
 		isActive && iconColor === 'border' ? 'selected' : iconColor;
 
 	return (
-		<Flex as="span" flexDirection="column" alignItems="center">
+		<Flex
+			alignItems="center"
+			as="span"
+			css={{
+				gridColumnStart: 1,
+				gridRowStart: 1,
+				height: '100%',
```

Comment:
Is this unnecessary?

Reply:
Oh yeah, I totally forgot to clean all this up, thank you!

- [x] Remove all redundant CSS related to grid work